src/status_im/contexts/wallet/sheets/network_preferences/view.cljs

```diff
@@ -51,52 +68,80 @@
              :blur-amount 20
              :blur-radius 25}])
          [quo/drawer-top
-          {:title       (i18n/label :t/network-preferences)
-           :description (if watch-only?
-                          (i18n/label :t/network-preferences-desc-1)
-                          (i18n/label :t/network-preferences-desc-2))
+          {:title       (or title (i18n/label :t/network-preferences))
+           :description (when-not receiver?
+                          (if watch-only?
+                            (i18n/label :t/network-preferences-desc-1)
+                            (i18n/label :t/network-preferences-desc-2)))
            :blur?       blur?}]
-         [quo/data-item
-          {:status          :default
-           :size            :default
-           :description     :default
-           :label           :none
-           :blur?           blur?
-           :card?           true
-           :title           (i18n/label :t/address)
-           :custom-subtitle (fn []
-                              [quo/address-text
-                               {:networks current-networks
-                                :address  address
-                                :blur?    blur?
-                                :format   :long}])
-           :container-style (merge style/data-item
-                                   {:background-color (colors/theme-colors colors/neutral-2_5
-                                                                           colors/neutral-90
-                                                                           theme)})}]
+         (when-not receiver?
```

Comment:
here we are using receiver? - I think it's nice if we can be consistent with our language across the domain. 
i.e we have to? in some places and receiver? in others.
Can we argue that these are separate concepts? If not we should have them aligned so it's easier for other devs to reason with.

Reply:
Agree, I refactored to use `receiver?` on all over the codebase for the sake of consistency